### PR TITLE
Run `flutter pub get` in example

### PR DESCRIPTION
**Pull request by nilsreichardt**
I used Flutter version `2.10.5`

### CODE REVIEW

File: example/pubspec.lock [hunk 1]
```diff
@@ -92,7 +92,14 @@ packages:
       name: matcher
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.12.10"
+    version: "0.12.11"
+  material_color_utilities:
```
Comment (yuhao900914) [inline example/pubspec.lock:96]: Is this dependency necessary?

Comment (yuhao900914) [inline example/pubspec.lock:96]: Is this dependency necessary?

Comment (nilsreichardt) [reply example/pubspec.lock:96]: It's a dependency by the Flutter SDK which have been added some months ago: https://github.com/flutter/flutter/pull/94377. I have not added any dependency. I just ran `flutter pub get` with the newest stable version in the example directory.

Comment (yuhao900914) [reply example/pubspec.lock:96]: Got it. Thanks.

**Pull request merged**